nixos/modules/services/misc/plikd.nix

```diff
@@ -0,0 +1,84 @@
+{ config, pkgs, lib, ... }:
+
+with lib;
+
+let
+  cfg = config.services.plikd;
+
+  format = pkgs.formats.toml {};
+
+  plikdCfg = format.generate "plikd.cfg" cfg.settings;
+in
+{
+  options = {
+    services.plikd = {
+      enable = mkEnableOption "Enable the plikd server";
+
+      user = mkOption {
+        type = types.str;
+        default = "plikd";
+        description = "User account under which Plikd runs.";
+      };
+
+      settings = mkOption {
+        type = format.type;
+        default = {
+          ListenPort    = 8080;
+          ListenAddress = "0.0.0.0";
```

Comment:
I'm assuming the intention is to put this behind a reverse proxy like `nginx`. If so, this seems like an odd default value. On the topic of reverse proxy... maybe you should document that to be the intention, or better yet write an optional integration with `nginx` right into this module.

Reply:
Good point I switch it to localhost